doc/source/defringe_examples.rst

```diff
@@ -299,3 +299,117 @@ G750M Observation of AGK+81D266
 .. image:: output_14_0.png
 
 
+Subarray Data
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    # Set up the data paths
+    sci_file = 'odqf11trq'
+    flat_file = 'odqf11060'
+
+
+.. code:: ipython3
+
+    # Normalize the contemporaneous flat field imagee
+    stistools.defringe.normspflat(f"{flat_file}_raw.fits",
+                                  f"{flat_file}_nsp.fits", do_cal=True,
+                                  wavecal=f"{sci_file}_wav.fits")
+
+.. parsed-literal::
+    File written:  /Users/stisuser/data/path/odqf11060_crj.fits
+
+.. code:: ipython3
+
+    # If using G750L data, set the order sorter fringes in the flat field equal to one.
+    # This particular dataset is G750L data.
+    flat_hdu = fits.open(f"{flat_file}_nsp.fits", mode="update")
+    flat_data = flat_hdu[1].data
+    flat_data[:, :250] = 1
+    flat_hdu.flush()
+    flat_hdu.close()
+
+.. code:: ipython3
+
+    # Pad the subarray data with zeros so that it is the same size as the flat field data
+    sci = fits.open(f"{sci_file}_crj.fits")
+    sci_data = sci[1].data
+    flat_data = fits.getdata(f"{flat_file}_nsp.fits")
+
+    ymin_sci = sci[0].header['CENTERA2'] - (sci[0].header['SIZAXIS2']/2.)
+    ymax_sci = sci[0].header['CENTERA2'] + (sci[0].header['SIZAXIS2']/2.)
+
+    full_sci = np.zeros(np.shape(flat_data), dtype='float32')
+    full_sci = [int(ymin):int(ymax), :] = sci_data
+    # Use the same process to populate the error and data quality arrays.
+
+    header1 = fits.getheader(sci)
```

Comment:
I haven't seen getheader work on opened FITS files before.  Maybe this instead?:
`getheader(sci)` --> `sci[0].header`
`fits.getheader(sci, 'SCI')` --> `sci['SCI'].header`